modules/spark/src/main/scala/notebook/spark/util/hack/SparkClosureUtilHack.scala

```diff
@@ -0,0 +1,63 @@
+package org.apache.spark.util.notebook.hack
+
+import scala.reflect.runtime.universe
+import org.apache.log4j.AppenderSkeleton
+import org.apache.log4j.spi.LoggingEvent
+
+trait SparkClosureUtilHack {
+  
+  private val CC = org.apache.spark.util.ClosureCleaner
+
+  def appendTo: String => Unit = println
+
+
+  private val YIPPEE = "Object cleaned and serializable!"
+  def clean(o:Object) = {
+    CC.clean(o, java.lang.Boolean.TRUE, java.lang.Boolean.TRUE)
+    // if we reach this point... it's good!
+    YIPPEE
+  }
+
+  private val logger:org.apache.log4j.Logger = {
```

Comment:
maybe call this a `closureUtilInternalLogger`, so it'd be more clear it's not just an arbitrary logger ?